editor/src/components/context-menu-items.ts

```diff
@@ -393,7 +387,7 @@ export const unwrap: ContextMenuItem<CanvasData> = {
 
 export const wrapInPicker: ContextMenuItem<CanvasData> = {
   name: 'Wrap in…',
-  shortcut: 'G',
+  shortcut: 'W',
```

Comment:
Is this shortcut just wrong then I take it?

Reply:
does not require _cmd_ to be held!